R/abort_if_file_doesnt_exist.R

```diff
@@ -0,0 +1,17 @@
+#' @examples
+#' path <- tempfile()
+#'
+#' abort_if_file_doesnt_exist(path)
+#' fs::file_create(path)
+#' abort_if_file_doesnt_exist(path)
+#'
+#' # Cleanup
+#' fs::file_delete(path)
+#' @noRd
+abort_if_file_doesnt_exist <- function(path) {
+  if (!fs::file_exists(path)) {
+    rlang::abort(glue::glue("This file must exist but it doesn't:\n{path}"))
+  }
+
+  invisible(path)
+}
```

Comment:
honestly, this seems overly dependent heavy... 3 dependent packages to achieve `stopifnot(file.exists(path))`?

Reply:
Good point. I refactored it to use only base R. See below.